src-tauri/build.rs

```diff
@@ -1,30 +1,9 @@
 fn main() {
-    let ffmpeg_link_mode = "static";
-
+    // Ensure the same macos_min_version is specified in `Package.swift`
     #[cfg(target_os = "macos")]
-    {
-        println!("cargo:rustc-link-search=native=/opt/homebrew/Cellar/zlib/1.3.1/lib/");
-        println!("cargo:rustc-link-search=native=/opt/homebrew/opt/x264/lib/");
-        println!("cargo:rustc-link-search=native=/opt/homebrew/opt/bzip2/lib/");
-        println!("cargo:rustc-link-lib=framework=CoreServices");
-        println!("cargo:rustc-link-lib=framework=CoreGraphics");
-        println!("cargo:rustc-link-lib=framework=CoreText");
-        println!("cargo:rustc-link-lib=framework=CoreFoundation");
-        println!("cargo:rustc-link-lib=framework=AudioUnit");
-        println!("cargo:rustc-link-lib=framework=AudioToolbox");
-        println!("cargo:rustc-link-lib=framework=CoreAudio");
-        println!("cargo:rustc-link-lib=framework=Security");
-        println!("cargo:rustc-link-lib=framework=VideoToolbox");
-
-        link("z", ffmpeg_link_mode);
-        link("x264", ffmpeg_link_mode);
-        link("bz2", ffmpeg_link_mode);
-    }
+    swift_rs::SwiftLinker::new("11")
```

Comment:
why "11" in const with understandable name.

Reply:
solved here: https://github.com/helmerapp/henx/commit/5ab39698957f7cfdc7098b6c05bdfc45d8ad31ba